CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 1.  [#2953](https://github.com/influxdata/chronograf/pull/2953): Fix error reporting in DataExplorer
 1.  [#2947](https://github.com/influxdata/chronograf/pull/2947): Fix Okta oauth2 provider support
 1.  [#2866](https://github.com/influxdata/chronograf/pull/2866): Change hover text on delete mappings confirmation button to 'Delete'
+1.  [#2919](https://github.com/influxdata/chronograf/pull/2919): Legacy Graph is missing type, default to line graph
```

Comment:
could you clarify this changelog entry into something a bit more straightforward, like `Make any graphs missing a graph type default to line` or `Automatically add graph type 'line' to any graph missing a type` ?

Reply:
👍